### PR TITLE
Integration test overhaul

### PR DESCRIPTION
**Pull request by mbwhite**
PR to resolve issues with the integration tests failing; slightly larger PR than ideal for the scale of the fix. However the original infrastructure used for the integration tests was a derivative of the (very) old byfn test network - and it was using old lifecycle and not updated in any recent time.

Therefore it seemed a manageable risk to remove it entirely and go with the modern solution of pulling in the single docker image of Microfab to use to spin up a real fabric network for testing. 

- additional tests are done with other deployments of Fabric in the fabric-tests repo
- microfab has all the features needed to support the 'integration/fv' testing of the chaincode, and is in one container

**Changes made:**
- Strand up Microfab rather than the multiple images
- Use a standard two org network as before (one peer per org so simpler than previously)
- The chaincode is built into a local repo inside each chaincode for testing
- The same tests are present with the same structure, minor changes made to support a slightly different approach

### CODE REVIEW

File: fabric-chaincode-integration-test/src/test/java/org/hyperleder/fabric/shim/integration/util/Command.java [hunk 1]
```diff
@@ -23,10 +25,17 @@ public class Command {
     protected List<String> cmd;
     protected Map<String, String> env;
 
-    Command(List<String> cmd) {
+    Command(List<String> cmd, Map<String,String> additionalEnv){
         this.cmd = cmd;
-        this.env = new HashMap<>();
+        // this.env = new HashMap(System.getenv());
```
Comment (jt-nti) [inline fabric-chaincode-integration-test/src/test/java/org/hyperleder/fabric/shim/integration/util/Command.java:30]: Did you mean to leave this commented out? Will not having the system env cause any problems?

File: fabric-chaincode-integration-test/src/test/java/org/hyperleder/fabric/shim/integration/util/Command.java [hunk 2]
```diff
@@ -23,10 +25,17 @@ public class Command {
...
+        this.env = new HashMap();
+        this.env.putAll(additionalEnv);
+    }
 
+    Command(List<String> cmd) {
+        this.cmd = cmd;
+        this.env = new HashMap();
+        // this.env = new HashMap(System.getenv());
```
Comment (jt-nti) [inline fabric-chaincode-integration-test/src/test/java/org/hyperleder/fabric/shim/integration/util/Command.java:38]: Did you mean to leave this commented out? Will not having the system env cause any problems?

**Review by jt-nti — APPROVED** [review]
A couple of minor comments and some concern about ccutil but definitely looks like an improvement, and hopefully admin apis may help further in the future

**Comment by mbwhite** [comment]
@mergify backport release-2.5

**Pull request merged**